### ISSUE TITLE
Scripts missing from 4.0.4 pypi release

### ISSUE THREAD

**Issue by toddrme2178** [jupyter/notebook#339]
The pypi version of notebook-4.0.4 is missing the "scripts" directory, meaning the scripts to launch the notebook are not installed.  The github version still has the directory.

**Comment by Carreau** [comment]
The scripts should be done by setup.py on [theses lines](https://github.com/jupyter/notebook/blob/656628cbe6ae9996b6fc30028322b85bcbb76137/setup.py#L182-L183) but maybe it make sens to ship the scripts in the sdist. 
I'm not sure.

**Comment by toddrme2178** [comment]
The scripts are not getting installed at all for me under 4.0.4.

**Comment by flacjacket** [comment]
I've also run into this, both here and with qtconsole.

**Comment by asergi** [comment]
I can confirm this bug as well.
Without the `scripts` directory the bin scripts are not generated. In fact:
```
% jupyter notebook            
jupyter: 'notebook' is not a Jupyter command
```
The GitHub version is not affetcted by this bug. Please update the PyPI tarball.
Thanks in advance.

**Comment by Carreau** [comment]
@minrk could this have been fixed by one of your recent changes in setup.py ?

**Comment by minrk** [comment]
Weeeird.... When I run sdist it includes them. I don't think this is fixed by a change in setup.py, but maybe something went wrong when building the last sdist. We have a bugfix release that we should be pushing out this week, I'll make sure it gets the scripts.

Installing with `pip` shouldn't need the scripts, since they aren't used when setuptools is involved.

**Comment by minrk** [comment]
Released 4.0.5 and verified that scripts are in the sdist.

**Comment by flacjacket** [comment]
From the PyPI upload [here](https://pypi.python.org/packages/source/n/notebook/notebook-4.0.5.tar.gz#md5=8d13252a5afaf07380d21cb408040d22) the scripts aren't in the package.

**Comment by minrk** [comment]
WTF, I tested and verified that it was working,

**Comment by minrk** [comment]
Ah, okay. I see that setuptools ruins upload. Fixing...

**Comment by minrk** [comment]
@flacjacket sorry about that, and thanks for catching it again. Fixed sdists for 4.0.5 are now on PyPI.

**Comment by flacjacket** [comment]
Great, thanks for the quick fix
